### DISCUSSION TITLE
Thank you for this project!

### DISCUSSION THREAD

**Discussion by Daniel15** [Dotnet-Boxed/Templates#279] [Q&A]
@RehanSaeed The contact page on your site (https://rehansaeed.com/contact/) is broken, so I thought I'd post here.

Thank you so much for this project! The "ASP.NET Core GraphQL Boxed" example in particular is extremely useful to learn how to structure an ASP.NET app that uses GraphQL. I couldn't find many other examples of this.

Feel free to close this issue, given it's not actually a bug 😛

**Comment by RehanSaeed** [comment]
Thanks for pointing out that my blog was broken. Now fixed.

Also, thanks for the feedback. Its always nice to hear that somebody found some work useful.
